app/controllers/api/v1/merchants/find_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::Merchants::FindController < ApplicationController
+
+  def show
```

Comment:
@jk1dd @caward12 That is a good way to go about the find method and I would continue down this road. 🎉